src/app/file-browser/file-list/file-list.component.scss

```diff
@@ -0,0 +1,182 @@
+@import "../../split-view/multi/terra-multi-split-view.component";
+
+:host
+{
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    display: flex;
+    flex-direction: column;
+}
+
+.loading-placeholder
+{
+    p
+    {
+        padding: 8px 0 9px 16px;
+        margin: 0;
+        display: inline-block;
+        color: $text-muted;
+    }
+    
+    .icon-loading
+    {
+        float: right;
+        font-size: 36px;
+    }
+}
+
+.terra-breadcrumbs
```

Comment:
ist das Überschreiben der Styles der Bread Crumbs notwendig?

Reply:
ja, ist an dieser Stelle notwendig, das das Context-Menu in diesem Fall anders positioniert werden muss, um auch innerhalb eines Overlays zu funktionieren. Im Normalfall würde das Context-Menu durch das Overlay abgeschnitten werden.